homeassistant/helpers/aiohttp_client.py

```diff
@@ -276,6 +281,7 @@ def _async_get_connector(
         ssl_context = False
 
     connector = aiohttp.TCPConnector(
```

Comment:
actually this doesn't work, due to the caching done within this method, specifically this code:

```python
key = DATA_CONNECTOR if verify_ssl else DATA_CONNECTOR_NOTVERIFY

if key in hass.data:
    return cast(aiohttp.BaseConnector, hass.data[key])
```